web/views.py

```
@@ -3788,6 +3793,154 @@ def get_object(self):
         return get_object_or_404(Storefront, store_slug=self.kwargs["store_slug"])
 
 
+def meetup_list(request: HttpRequest) -> HttpResponse:
+    meetups = Meetup.objects.select_related("creator").all().order_by("-created_at")
+
+    # Add annotation for user registration status when authenticated
+    if request.user.is_authenticated:
+        from django.db.models import Exists, OuterRef
+
+        meetups = meetups.annotate(
+            user_registered=Exists(MeetupRegistration.objects.filter(meetup=OuterRef("pk"), user=request.user))
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)